config/config.go

```diff
@@ -536,6 +536,8 @@ type MempoolConfig struct {
 	WalPath   string `mapstructure:"wal_dir"`
 	Size      int    `mapstructure:"size"`
 	CacheSize int    `mapstructure:"cache_size"`
+	// Maximum size of all txs in the mempool in bytes
```

Comment:
It can be a bit confusing as it can be understood as max tx size. Maybe instead "Maximum size of the mempool in bytes" or "Maximum size of the mempool list in bytes" in case we want to differentiate it from the mempool cache.

Reply:
> mempool list

users don't know anything about mempool list. I will change it to "Maximum size of the mempool in bytes"